cypress/support/constants.js

```diff
@@ -172,10 +172,10 @@ export const RECEIVING_WORKFLOW_NAMES = {
 };
 
 export const ACQUISITION_METHOD_NAMES = {
-  APPROVAL_PLAN: 'Approval plan',
```

Comment:
It's bad idea to rename constants. They can be used for creating orders. Let's create new constant like ACQUISITION_METHOD_NAMES_IN_MAPPING_PROFILES

Reply:
fixed